lib/streamlit/elements/vega_charts.py

```diff
@@ -215,6 +266,26 @@ def id_transform(data) -> dict[str, str]:
     return chart_dict
 
 
+def _check_spec_for_selections(spec: dict[str, Any]) -> None:
```

Comment:
should we use a type-alias for specs? 

```python
type VegaSpec = dict[str, Any]
```

Just to make it more clear what kind of spec we expect

Reply:
Good idea, added it 👍